images/builder/install-protoplugins.sh

```diff
@@ -8,8 +8,15 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
-# 8ba23be9613c672d40ae261d2a1335d639bdd59b == tag: cmd/protoc-gen-go-grpc/v1.3.0
-go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@8ba23be9613c672d40ae261d2a1335d639bdd59b
+# renovate: datasource=github-tags depName=grpc/grpc-go
+GRPC_VERSION=cmd/protoc-gen-go-grpc/v1.3.0
+
+GRPC_VERSION=${GRPC_VERSION#cmd/protoc-gen-go-grpc/}
+go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@${GRPC_VERSION}
+
+# renovate: datasource=github-releases depName=protocolbuffers/protobuf-go
 go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.30.0
-go install github.com/mitchellh/protoc-gen-go-json@v1.1.0
+# renovate: datasource=github-releases depName=mfridman/protoc-gen-go-json
+go install github.com/mfridman/protoc-gen-go-json@v1.2.0
```

Comment:
wait, this should be 1.1.0

Reply:
Did you change to v1.2.0 because of the same reason mentioned in https://github.com/cilium/cilium/pull/37281? The v1.1.0 also exists in github.com/mfridman/protoc-gen-go-json

Reply:
Yep, I had to change it because it doesn't exist in the `mfridman` fork: https://github.com/mfridman/protoc-gen-go-json/commit/1238e4c97711a697daeec2f4099585c284290bbd

```
[ backporter's notes: changed the version to v1.2.0 as it is the
  first one from the mfridman fork. ]
```